src/content/docs/en/recipes/tailwind-rendered-markdown.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Style Rendered Markdown with Tailwind Typography
+description: Learn how to use @tailwind/typography to style your rendered Markdown
+i18nReady: true
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+
+You can use [Tailwind](https://tailwindcss.com)'s Typography plugin to style rendered Markdown from sources such as Astro's [**content collections**](/en/guides/content-collections/).
+This recipe will teach you how do create a reusable Astro component to **make your Markdown shine!**
+
+## Prerequisites
+
+An Astro project that:  
+
+	- is [initialized with Tailwind](/en/guides/integrations-guide/tailwind/),  
+	- is set up to use **content collections** or another Markdown rendering system.
+
+## Setting Up `@tailwindcss/typography`
+
+First, install `@tailwindcss/typography` using your preferred package manager.
+
+<PackageManagerTabs>
+ 	<Fragment slot="npm">
+	```shell 
+	npm install -D @tailwindcss/typography
+	```
+	</Fragment>
+  	<Fragment slot="pnpm">
+	```shell 
+	pnpm install -D @tailwindcss/typography
+	```
+	</Fragment>
+  	<Fragment slot="yarn">
+	```shell
+	yarn add --dev @tailwindcss/typography
+	```
+	</Fragment>
+</PackageManagerTabs>
+
+Then, add the package as a plugin in your Tailwind configuration file.
+
+```diff lang="js"
+// tailwind.config.js
+/** @type {import('tailwindcss').Config} */
+
+module.exports = {
```

Comment:
Replace with `export default defineConfig = {`

Reply:
I do believe that the proper ESM syntax for this export would be to just export an object like so:
```diff
/** @type {import('tailwindcss').Config} */
- module.exports = {
+ export default {
  // ...
}
```

I could definitely fix that, since the Tailwind plugin for Astro generates a config file using ESM like I show above :)

Reply:
I included this in my editing pass. Please correct if needed!